test/integration/includes.cc

```diff
@@ -55,8 +55,6 @@ TEST(IncludesTest, Includes)
 
   sdf::Root root;
   sdf::Errors errors = root.Load(worldFile);
-  for (auto e : errors)
-    std::cout << e.Message() << std::endl;
   EXPECT_TRUE(errors.empty());
```

Comment:
It's useful to print the errors when the expecation fails.
```suggestion
  EXPECT_TRUE(errors.empty()) << errors;
```